osi_featuredata.proto

```diff
@@ -104,13 +113,20 @@ message SensorDetectionHeader
     //
     optional uint32 number_of_valid_detections = 6;
 
-    // The ID of the sensor at host vehicle's mounting_position.
+    // The ID of the sensor at host vehicle's \c #mounting_position.
     //
     // This ID can equal \c SensorData::sensor_id, if \c SensorData holds only
     // data from one sensor/sensor model.
     //
     optional Identifier sensor_id = 7;
 
+    // The extended qualifier describes the reason (not the effect) why the
+    // event data qualifier \c #data_qualifier is reduced or not available.
```

Comment:
[...] event data, indicated by the qualifier \c # data_qualifier, is reduced or not available.